### DISCUSSION TITLE
Databend for OLTP web applications.

### DISCUSSION THREAD

**Discussion by ansarizafar** [databendlabs/databend#3357] [Q&A] [answered]
Is there a plan to make databend suitable for OLTP web applications?

**Comment by bohutang** [comment, marked as answer]
> Is there a plan to make databend suitable for OLTP web applications?

Hi,

What's the `OLTP web applications` requirements?
If it's data-intensive applications(OLAP-based) i think it's no problem.

**Comment by Xuanwo** [comment]
@BohuTANG Let's move question alike issues to the [discussion](https://github.com/datafuselabs/databend/discussions) Q&A instead.

**Comment by Xuanwo** [comment]
I think all issues under `C-quesion` should be moved either.

Here is the query: https://github.com/datafuselabs/databend/issues?q=is%3Aopen+is%3Aissue+label%3AC-question

**Comment by ansarizafar** [comment]
OLTP enables the real-time execution of large numbers of transactions by large numbers of people, whereas online analytical processing (OLAP) usually involves querying these transactions (also referred to as records) in a database for analytical purposes. https://www.oracle.com/database/what-is-oltp/

Currently, databend can't be used to update or delete rows. Documentation shows only select and insert statement under data manipulation language section. My understanding is that databend is for data analytics only like clickhouse and not suitable for using as a general purpose RDMS,. Am I right?

Let me rephrase my question, Is it possible to replace mysql or postgres with databend for using as a main back end data store for an eCommerce or ERP web application?

**Comment by ansarizafar** [comment]
I am waiting for a reply.

**Reply by Xuanwo** [reply]
Hi, thanks for waiting.

We have not plan to support OLTP workload so far. It's possible to use databend if your application is data-intensive,  but it's not the main goal of this project. 

> Databend is an elastic and reliable Cloud Data Warehouse.
